templates/support/SecurityClient.swift.hbs

```diff
@@ -0,0 +1,361 @@
+//  
+//  {{>frag/generatedBy}}
+//
+
+import Foundation
+
+public protocol SecurityClient {
+    /** An API request received a 401 response */
```

Comment:
Let's expand this documentation, taking into account my earlier renaming suggestion if appropriate. I think we want to document that the session will have been reauthenticated when this method returns, otherwise it throws an error.